contracts/dapis/DapiServer.sol

```diff
@@ -806,6 +811,145 @@ contract DapiServer is
         );
     }
 
+    ///                     ~~~OEV~~~
+
+    /// @notice Updates an OEV proxy Beacon using data signed by the respective
+    /// Airnode, without requiring a request or subscription
+    /// @param airnode Airnode address
+    /// @param templateId Template ID
+    /// @param timestamp Timestamp used in the signature
+    /// @param data Response data (a `uint256` encoded in contract ABI)
+    /// @param metadata Metadata that the OEV proxy will use (e.g., auction bid
+    /// parameters)
+    /// @param signature Template ID, a timestamp, response data and metadata
+    /// signed by the Airnode address
+    function updateOevProxyBeaconWithSignedData(
+        address airnode,
+        bytes32 templateId,
+        uint256 timestamp,
+        bytes calldata data,
+        bytes calldata metadata,
+        bytes calldata signature
+    ) external override onlyValidTimestamp(timestamp) {
+        require(
+            (
+                keccak256(
+                    abi.encodePacked(templateId, timestamp, data, metadata)
+                ).toEthSignedMessageHash()
+            ).recover(signature) == airnode,
+            "Signature mismatch"
+        );
+        bytes32 beaconId = deriveBeaconId(airnode, templateId);
+        uint224 updatedBeaconValue = decodeFulfillmentData(data);
+        require(
+            timestamp >
+                oevProxyAddressToIdToDataFeed[msg.sender][beaconId].timestamp,
```

Comment:
I'm trying to think of a scenario where also checking `dataFeeds[dataFeedId].timestamp` might be relevant but it doesn't seem like it is

Reply:
Nah